posters-overrides.json

```diff
@@ -5613,7 +5613,13 @@
    "number": 1975
   },
   {
-   "number": 1976
+   "number": 1976,
+   "title": "Re-Executability Assessment of the Recent Autism Literature",
+   "institution": "University of Massachusetts Medical School",
+   "presenter": "David Kennedy",
+   "categories": "Neuroinformatics and Data Sharing<br>Informatics Other",
+   "videochat": "<a href=\"https://meet.jit.si/ohbm2020-1976\" target=\"_ohbm2020-1976\">jitsi:ohbm2020-1976</a>",
```

Comment:
full record wasn't needed, only fields to add... I will merge and remove the rest